go/chat/server.go

```diff
@@ -1213,6 +1219,11 @@ func (h *Server) PostEditNonblock(ctx context.Context, arg chat1.PostEditNonbloc
 	parg.IdentifyBehavior = arg.IdentifyBehavior
 	parg.OutboxID = arg.OutboxID
 	parg.Msg.ClientHeader.MessageType = chat1.MessageType_EDIT
+	// Sending device may have changed
+	parg.Msg.ClientHeader.SenderDevice, err = h.getGregorDeviceID()
```

Comment:
Can just delete all the setting of device ID here, I think this code is old and we set this later on.

Reply:
for a superseding message? I don't think so but I will try it

Reply:
ok it works without that, thanks